poetry/console/commands/add.py

```diff
@@ -100,7 +100,7 @@ def handle(self):
         packages = [name for name in packages if name not in existing_packages]
 
         if not packages:
-            self.poetry.file.write(content)
+            self.poetry.file.write(original_content)
```

Comment:
Should we just remove this as we do not modify anything before this?

Reply:
yes, think so, tests seem to work anyway...